### PR TITLE
LPS-120033 Update @clayui/* packages to new version

### PR DESCRIPTION
**Pull request by matuzalemsteles**
After commerce entered the master, there were some outdated clay v3 packages, I ended up changing so we don't have old packages and different versions in yarn.lock, but there are still some changes that seem to be caused by the package entry in yarn.lock.

- Diff https://github.com/liferay/clay/pull/3667

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
ci:test:sf

**Comment by matuzalemsteles** [comment]
ci:test:relevant

**Comment by matuzalemsteles** [comment]
Some tests were breaking because the commerce has a version of React more updated than the rest of the portal, send a commit to leave the same version with the rest of the portal. I'm not sure if this will have an impact on commerce. Run the tests again

**Comment by matuzalemsteles** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
I checked and most of the additional deps come from the new commerce modules, so we can make a pass at them next week.

Since this is green, going to forward it to get all clay and react modules in sync and then we can keep pushing! 👍

**Comment by jbalsas** [comment]
ci:forward

File: modules/yarn.lock [hunk 1]
```diff
@@ -5842,6 +5853,11 @@ clay@2.18.1:
   resolved "https://registry.yarnpkg.com/clay/-/clay-2.18.1.tgz#c13bacd70d3e12b4da26b57d3506eb5046224f2d"
   integrity sha512-BDCpIpMjjgwHh3hnjt3UtIYaHvi41of5HmcL8KqMuMIy8KmzLM/7NDVphxU0G7IjisLS6RqTKL4gDlrss6xTyw==
 
+clay@^2.4.1:
+  version "2.21.4"
+  resolved "https://registry.yarnpkg.com/clay/-/clay-2.21.4.tgz#854a645ebe78eeaadd23660a060a0c5851639d44"
+  integrity sha512-k9rtzpBH2Dr0k0uiA20+EJQJl5CNoSjBruMlJQtWIhvSL4vmFzjz3fFL6yQD91OKCDdJYK7WkaTflbIX7TEDwQ==
+
```
Comment (wincent) [inline modules/yarn.lock:5860]: This one is a bit concerning (2.4.1 vs 2.18.1) ie. 15 Nov 2018 vs 10 Sep 2019 🙏 

The d3 dupes look relatively innocuous.

Comment (jbalsas) [reply modules/yarn.lock:5860]: Yeah, the thing is they're mostly `devDeps` which I have no idea how are passing SF... so I figured we'll take a nice time this week to sort out the whole commerce integration to check for anomalies like this.

Comment (wincent) [reply modules/yarn.lock:5860]: Probably these suppressions:

- https://github.com/liferay/liferay-portal/blob/06c02ecf4e21e01838db58ef8455a9710d0170d4/modules/apps/commerce/source-formatter-suppressions.xml#L83-L89
- https://github.com/liferay/liferay-portal/blob/06c02ecf4e21e01838db58ef8455a9710d0170d4/modules/dxp/apps/commerce/source-formatter-suppressions.xml#L10-L12

**Comment by rdai10** [comment]
Will the infra team also backport this change to 7.2.x branch? The introduction of a different react and react dom version is also causing CI test failures for IS project on 7.2.x.
For reference: https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.2.x)/AXIS_VARIABLE=0,label_exp=!master/24072/console
Source code: https://github.com/liferay/liferay-portal-ee/tree/7.2.x/modules/dxp/apps/osb/osb-provisioning/osb-provisioning-web/

I'm not sure what the proper course of action should be as it involves the commerce team and their code base. Please let me know, thanks!

**Comment by jbalsas** [comment]
Hey @rdai10, @john-co, sent https://github.com/brianchandotcom/liferay-portal-ee/pull/31437 to help commerce out. Confirmed with @marco-leo that Commerce is not meant to work in that branch as is, so changes should be fine as long as they remove the dependency redundancies.

**Pull request closed without merging**